Datahub.Core/Layout/ErrorModal.razor

```diff
@@ -56,55 +61,73 @@
         ApiService.LastException = null;
         Hide = true;
         navigationManager.NavigateTo("/");
-    }  
+    }
 
     public RenderFragment GenerateExceptionMessage(Exception ex, int level = 1)
     {
         RenderFragment displaFragment = b =>
         {
             b.OpenElement(level, "p");
-            b.OpenElement(level+1, "b");
-            b.AddContent(level+2, Localizer["ERROR-PAGE.Message-Label"]);
+            b.OpenElement(level + 1, "b");
```

Comment:
you don't need to write code in c# - for a render fragment you can directly write razor
RenderFragment f = <p><b>@Localizer[...]

Reply:
Looks like Github ate your HTML code. It should look something like this:
`RenderFragment blah = @<div>...</div>;`

Reply:
I am going to place an Exception display component, that just discovered can be recursive. Meaning it can refer to itself.

Reply:
That didn't work. So for now we have to live with the ugly fragment code. It was there from before, I just refactored the code around it.